doc/classes/Node3D.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Node3D" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Most basic 3D game object, parent of all 3D-related nodes.
+		Base object in 3D space, inherited by all 3D nodes.
 	</brief_description>
 	<description>
-		Most basic 3D game object, with a [Transform3D] and visibility settings. All other 3D game objects inherit from [Node3D]. Use [Node3D] as a parent node to move, scale, rotate and show/hide children in a 3D project.
-		Affine operations (rotate, scale, translate) happen in parent's local coordinate system, unless the [Node3D] object is set as top-level. Affine operations in this coordinate system correspond to direct affine operations on the [Node3D]'s transform. The word local below refers to this coordinate system. The coordinate system that is attached to the [Node3D] object itself is referred to as object-local coordinate system.
-		[b]Note:[/b] Unless otherwise specified, all methods that have angle parameters must have angles specified as [i]radians[/i]. To convert degrees to radians, use [method @GlobalScope.deg_to_rad].
-		[b]Note:[/b] Be aware that "Spatial" nodes are now called "Node3D" starting with Godot 4. Any Godot 3.x references to "Spatial" nodes refer to "Node3D" in Godot 4.
+		The [Node3D] node is the base representation of a node in 3D space. All other 3D nodes inherit from this class.
+		Affine operations (translation, rotation, scale) are calculated in the coordinate system relative to the parent, unless the [Node3D]'s [member top_level] is [code]true[/code]. In this coordinate system, affine operations correspond to direct affine operations on the [Node3D]'s [member transform]. The term "[i]parent space[/i]" refers to this coordinate system. The coordinate system that is attached to the [Node3D] itself is referred to as object-local coordinate system, or "[i]local space[/i]".
+		[b]Note:[/b] Unless otherwise specified, all methods that need angle parameters must receive angles in [i]radians[/i]. To convert degrees to radians, use [method @GlobalScope.deg_to_rad].
+		[b]Note:[/b] In Godot 4, "[i]Spatial[/i]" nodes are named "[i]Node3D[/i]". Some parts of the documentation may still mention Godot 3.x's "[i]Spatial[/i]" nodes.
```

Comment:
```suggestion
		[b]Note:[/b] In Godot 3, [Node3D] was called "Spatial".
```

I don't like keeping notes like this around, nearly 2 years and 4 minor versions into Godot 4's lifespan. At the very least, we should remove the part about parts of the documentation being incorrect, and we should treat Godot 4 as the norm rather than Godot 3.

It looks like all(?) the remaining references to "spatial" in the class reference were actually recently fixed in one of your PRs. So, good job on that and we *definitely* should not say that our docs are wrong, in our docs.

Reply:
I'd remove this line too but, the n.1 fear of these kinds of PRs is the loss of any possible information that may have solved some old, perhaps overly-specific issue opened in the past.

Reply:
I think adding `spatial` as a keyword in Node3D would actually make this note even more redundant.

Reply:
I think keeping the note around in general is justified (a rename of a core class in a major version is a big change), but we should not explicitly mention that our docs still use the wrong terms. The issue seems generally fixed in the class reference. I haven't checked the manual yet, but in my experience I haven't seen usage of spatial, and if there are still some usages I can make a PR to correct them.